packages/vm/testdata/contract.wasm

```diff
@@ -1 +1 @@
-contract_0.7.wasm
\ No newline at end of file
+contract_0.8.wasm
```

Comment:
Huh? How does this work in `include_bytes!()`?
Does that do a redirect?

I do agree with now testing with v0.7 and v0.8 and having a generic "contract" which is always the most recent version.

Reply:
This is a symlink in the file system. include_bytes reads it the same way as any other file